### ISSUE TITLE
missing title attribute in cal frame

### ISSUE THREAD

**Issue by hichemfantar** [calcom/cal.com#15945]
### Issue Summary

the lack of the title attribute negatively affects accessibility and lighthouse scores.

### Steps to Reproduce

1. run the lighthouse test on any site that embeds cal.com

### Expected behavior

title attribute should be present

### Screenshots

![image](https://github.com/user-attachments/assets/2328a887-8b78-4f2f-92ab-3d7c66945ec6)

### Environment

- [x] app.cal.com
- [ ] Self-hosted Cal

### Desktop (please complete the following information)

```markdown
- Browser: Chrome
```
